app/Modules/Invoices/Controllers/InvoiceEditController.php

```diff
@@ -34,6 +34,7 @@ class InvoiceEditController extends Controller
     public function edit($id)
     {
         $invoice = Invoice::with(['items.amount.item.invoice.currency'])->find($id);
+        $invoiceItemsCount = isset($invoice->invoiceItems) ? count($invoice->invoiceItems) : 0;

```

Comment:
Please stick to a uniform variable naming. Use snake case for single vars: `$invoice_item_count`

Reply:
Resolved in 82b2a34